src/gluonts/model/deepar/_estimator.py

```diff
@@ -111,6 +111,13 @@ class DeepAREstimator(GluonEstimator):
     num_parallel_samples
         Number of evaluation samples per time series to increase parallelism during inference.
         This is a model optimization that does not affect the accuracy (default: 100)
+    imputation_method
+        Select the method to replace the missing values.
+        - "standard" to just replace them with the dummy_value
+        - "mean" to replace them with the mean
+        - "median" to replace them with the median
+        - "last_val" to replace them with the last non missing value
+        (default: "standard")
```

Comment:
I think repeating this docstring for each estimator may not scale well (eg in case other options are added). Maybe we could mention the role of this option and the default value, and point to the docs for AddObservedIndicator to know more about the available options. What do you think?

Reply:
Yeah sure, I was not sure how to do that best.

Sounds good, will change this.